spec/factories/state_file_nj_intakes.rb

```diff
@@ -98,6 +98,8 @@
         qualifying_widow: 5,
       }[evaluator.filing_status.to_sym] || evaluator.filing_status
       intake.direct_file_data.filing_status = numeric_status
+      intake.direct_file_data.primary_ssn = evaluator.primary_ssn || intake.direct_file_data.primary_ssn
+      intake.direct_file_data.spouse_ssn = evaluator.spouse_ssn || intake.direct_file_data.spouse_ssn
```

Comment:
I'm not entirely sure why these lines are necessary, but it's what allows me to pass in an SSN to the factory create and have it actually come back with the SSN I gave it. Without these lines, the SSN will get overridden by whatever is in the raw DF data XML on line 88 here.

Reply:
I wonder if we can do this for all of the fields rather than individually?

Reply:
so, this could have something to do with the fact that we usually read ssn like this `intake.primary.ssn` which you can see the inner workings of [here](https://github.com/codeforamerica/vita-min/blob/main/app/models/state_file_base_intake.rb#L199). i just realized the nj intake has a `primary_ssn` column whereas our other state file intake models do not. are you collecting ssn from the client or planning to get it from direct file?

Reply:
related but probably out of scope for this PR: this made me realize that the nj intake has a `primary_ssn` column whereas our other state file intake models do not. are you collecting ssn from the client or planning to get it from direct file?

Reply:
note to self - will circle back to these factory suggestions in a future commit that will remove primary_ssn and spouse_ssn fields altogether from the data model